engines/suppliers/Gemfile.lock

```diff
@@ -2,68 +2,93 @@ PATH
   remote: .
   specs:
     suppliers (0.0.1)
-      rails (~> 4.1.7)
+      rails (~> 4.2.5)
```

Comment:
Sorry my bad, neither Gemfile nor Gemfile.lock should be commited for engines, because they are simply gems.